src/index.json

```diff
@@ -653,7 +653,7 @@
         "signalr": [
             {
                 "filename": "signalr-0.1.0-py2.py3-none-any.whl",
-                "sha256Digest": "0cd98580aa10b660602c1b4496c9beef24cf9c8e15df8113b2642c75457772ee",
+                "sha256Digest": "be38f9e49f1a92d194e7f141d978e10de852f77b63db2ef6c02f838fc3067f9f",
                 "downloadUrl": "https://azureclisignalr.blob.core.windows.net/cli-extensions/signalr-0.1.0-py2.py3-none-any.whl",
```

Comment:
Publish with a higher version number.

Reply:
This is causing CI to fail for this PR https://github.com/Azure/azure-cli-extensions/pull/163